ckanext/aircan/tests/test_plugin.py

```
@@ -0,0 +1,56 @@
+To temporary patch the CKAN configuration for the duration of a test you can use:
+
+    import pytest
+
+    @pytest.mark.ckan_config("ckanext.myext.some_key", "some_value")
+    def test_some_action():
+        pass
+"""
+import ckanext.aircan.plugin as plugin
+
+
+@pytest.mark.ckan_config("ckan.plugins", "aircan")
```

Ruff (0.14.10)

Undefined name `pytest`

(F821)